internal/service/luhn.go

```diff
@@ -0,0 +1,46 @@
+package service
+
+import (
+	"errors"
+)
+
+const (
+	asciiZero = 48
```

Comment:
Эту константу можно не создавать:

```suggestion
	asciiZero = '0'
```